adagrams/game.py

```diff
@@ -1,13 +1,152 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J","X"],
+    10: ["Q","Z"]  
+}
+
+def get_draw_pool():
+    draw_pool = []
+
+    for letter, num in LETTER_POOL.items():
+        while num > 0:
+            draw_pool.append(letter)
+            num -= 1
+            # print(num)
+    return draw_pool
+
+# print(get_draw_pool())
+
+
 def draw_letters():
-    pass
+    get_letter = []
+    draw_pool = get_draw_pool()
+
+    for _ in range(10):
+        r = randint(0, len(draw_pool)-1)
```

Comment:
Operators like `-` should be surrounded with white spaces. Take a look at the [PEP8 style guide on white spaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).  In industry, code that doesn't follow an agreed upon code styles will need to be revised to ensure the codebase is neat and consistent.
```suggestion
        r = randint(0, len(draw_pool) - 1)
```